src/style.css

```diff
@@ -0,0 +1,83 @@
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.form {
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+.form:hover {
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.inputBig::placeholder {
+  font-size: 16px;
+}
+
+.inputSmall::placeholder {
+  font-size: 14px;
+}
+
+.input {
+  width: 100%;
+  font-weight: 300;
+  font-family: inherit;
+  border: none;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 400;  
```

Comment:
Text should be more weighted
<img width="708" alt="Screenshot 2022-01-18 at 16 27 51" src="https://user-images.githubusercontent.com/32794096/149956398-39a90a3e-d5f5-4aa4-8912-91088f585eef.png">